forge-gui/res/cardsfolder/upcoming/monster_manual_zoological_study.txt

```diff
@@ -0,0 +1,16 @@
+Name:Monster Manual
+ManaCost:3 G
+Types:Artifact
+A:AB$ ChangeZone | Cost$ 1 G T | Origin$ Hand | Destination$ Battlefield | ChangeType$ Creature | ChangeNum$ 1 | SpellDescription$ You may put a creature card from your hand onto the battlefield.
+AlternateMode:Adventure
+Oracle:{1}{G}, {T}: You may put a creature card from your hand onto the battlefield.
+
+ALTERNATE
+
+Name:Zoological Study
+ManaCost:2 G
+Types:Sorcery Adventure
+A:SP$ Mill | Cost$ 2 G | Defined$ You | NumCards$ 5 | RememberMilled$ True | SubAbility$ DBChangeZone | SpellDescription$ Mill five cards, then return a creature card milled this way to your hand.
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Hand | Hidden$ True | ChangeType$ Creature.IsRemembered | ChangeTypeDesc$ creature cards milled this way | ChangeNum$ 1 | SelectPrompt$ Choose a creature card milled this way | SubAbility$ DBCleanup
```

Comment:
I think this might need `Mandatory$ True` - see if you can skip it even when you mill a creature. You shouldn't be able to.

If you don't mill a creature, `Mandatory$ True` won't break anything.